From 069e729708202a532c7f584ee44a311441ab517b Mon Sep 17 00:00:00 2001
From: Yan Zhao <yan.y.zhao@intel.com>
Date: Mon, 26 Oct 2020 14:05:30 +0800
Subject: [PATCH] acpi proxy: relocate ACPI_PROXY_BASE

the original range (0x7FD7F000 - 0x7FD85FFF)
conflicts with range (0x7FC00000 - 0x7FDFFFFF)
EFIBootServiceData(4) and would cause PAGE FAULT in
flashing bootloader phase.

Signed-off-by: Yan Zhao <yan.y.zhao@intel.com>
---
 include/hw/misc/acpi_proxy.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/hw/misc/acpi_proxy.h b/include/hw/misc/acpi_proxy.h
index aa48d4b196..71125c5c0d 100644
--- a/include/hw/misc/acpi_proxy.h
+++ b/include/hw/misc/acpi_proxy.h
@@ -3,7 +3,7 @@

 #include "qom/object.h"

-#define ACPI_PROXY_BASE         0x7fd7f000
+#define ACPI_PROXY_BASE         0x808000
 #define ACPI_PROXY_LEN          0x7000

 #define TYPE_ACPI_PROXY "acpi-proxy"
--
2.17.1

